appinventor/components/src/com/google/appinventor/components/runtime/Slider.java

```diff
@@ -69,6 +73,23 @@
   private final static int initialLeftColor = Component.COLOR_ORANGE;
   private final static String initialLeftColorString = Component.DEFAULT_VALUE_COLOR_ORANGE;
 
+  // seekbar.getThumb was introduced in API level 16 and the component warns the user
+  // that apps using Sliders won't work if the API level is below 16.  But for very old systems the
+  // app won't even *load* because the verifier will reject getThumb.  I don't know how old - the
+  // rejection happens on Donut but not on Gingerbread.
+  // The purpose of SeekBarHelper class is to avoid getting rejected by the Android verifier when the
+  // Slider component code is loaded into a device with API level less than Gingerbread.
+  // We do this trick by putting the use of getThumb in the class SeekBarHelper and arranging for
+  // the class to be compiled only if the API level is at least Gingerbread.  This same trick is
+  // used in implementing the Sound component.
+  private class SeekBarHelper {
+    public void getThumb(int alpha) {
+      seekbar.getThumb().mutate().setAlpha(alpha);
+    }
+  }
+
+  public final boolean referenceGetThumb = (SdkLevel.getLevel() >= SdkLevel.LEVEL_GINGERBREAD);
```

Comment:
@halatmit I'm going to change LEVEL_GINGERBREAD to LEVEL_JELLYBEAN (that's the magic!).